disnake/role.py

```diff
@@ -536,14 +602,39 @@ async def edit(
             await self._move(position, reason=reason)
 
         payload: Dict[str, Any] = {}
-        if color is not MISSING:
-            colour = color
 
-        if colour is not MISSING:
-            if isinstance(colour, int):
-                payload["color"] = colour
+        colors: Dict[str, Any] = {}
+        if color is not MISSING or colour is not MISSING:
+            primary_colour = color or colour
```

Comment:
```suggestion
        if color is not MISSING:
            primary_colour = color
        elif colour is not MISSING:
            primary_colour = colour
```
This would not allow setting `color = 0` otherwise